newIDE/app/src/MainFrame/index.js

```diff
@@ -1882,8 +1954,13 @@ const MainFrame = (props: Props) => {
         },
         editorTabs
       );
+      setIsLoadingProject(false);
```

Comment:
Any reason for adding this?

Reply:
Yes, it was previously done in the `openLayout` method and I don't think it was its responsibility to close the loader. For instance when opening a project with the option `openAllScenes`, it would set the loader many times to false. So I added them after `openLayout` has been called

Reply:
sounds good then!